tests/conftest.py

```diff
@@ -231,6 +232,13 @@ def _subprocess_wrapper():
 
 @pytest.hookimpl(tryfirst=True)
 def pytest_runtest_protocol(item):
+    if item.get_closest_marker("skip"):
```

Comment:
Can we add a comment on why we check the `skip` marker first

Reply:
oh there's no particular reason apart from checking the simplest case first.